R/cor_phylo_info.R

```diff
@@ -0,0 +1,206 @@
+# Functions for extracting info from cor_phylo call
```

Comment:
For the functions within this file, if it can be used for other functions except `cor_phylo`, can you move it to the `utils.R`? Then for the remaining functions, can you move them to the `cor_phylo.R`? Then we can delete this file.
If you can write some brief documentation for each function, that would be great! Thanks!